querydsl-jpa/src/test/java/com/querydsl/jpa/AbstractJPATest.java

```diff
@@ -319,6 +319,14 @@ public void Case3() {
     }
 
     @Test
+    @ExcludeIn(MYSQL) // doesn't work in Eclipselink
+    public void Case4() {
+        NumberExpression<Float> numExpression = cat.bodyWeight.floatValue().divide(otherCat.bodyWeight.floatValue()).multiply(100);
+        NumberExpression<Float> numExpression2 = cat.id.when(0).then(0.0F).otherwise(numExpression);
+        query().from(cat, otherCat).list(numExpression2);
```

Comment:
Shouldn't these tests make assertions or are you just assuming that if an exception isn't thrown it's working?